lib/checks/keyboard/focus.js

```diff
@@ -0,0 +1,7 @@
+if (node.getAttribute('onFocus') === 'this.blur()') {
+	return false;
+} else if (node.getAttribute('onFocus').indexOf('this.blur()') > -1) {
+	return undefined;
+} else {
+	return true;
+}
```

Comment:
I would lose the `if ... else if ... else` and `return` as is.
Each of the code block returns either way at the moment.

Reply:
Unsure how to write this into an single return statement, do you have an example of this?

Reply:
Something like this maybe? The two if blocks could be combined potentially, but since they return different results this probably works fine.

```
if (node.getAttribute('onFocus') === 'this.blur()') {
	return false;
} 
if (node.getAttribute('onFocus').indexOf('this.blur()') > -1) {
	return undefined;
}
return true;
```